truffle-config.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
   build: {},	
   compilers: {	
     solc: {	
-      version: '0.5.16',
+      version: '0.6.6',
```

Comment:
We dont need more networks, this is only used for the flattener script.
Anyway this can be added in a future with no need to change the contracts sourcecode